src/lib/pipeline.ml

```diff
@@ -324,9 +328,11 @@ module Construct = struct
        make_target }
       bam_pair
 
-end
+  let rna_hla_typer fastq_sample = Rna_hla_typer fastq_sample
```

Comment:
To be specific, this new module (`Pipeline.Specific`) should probably be very simple redirects:

```OCaml
let mutect = Mutect.pipeline
let muse = Muse.pipeline
...
```